### DISCUSSION TITLE
[Feature] Support multiple server connection switches

### DISCUSSION THREAD

**Discussion by Triple-Z** [immich-app/immich#10590] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Just like Swift Paperless.

https://github.com/paulgessinger/swift-paperless/issues/72

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#308
